data/human/transport missions.txt

```diff
@@ -3582,3 +3582,743 @@ mission "Hauler VI cargo prototype 3"
+				decline
+	on visit
+		dialog `The filming equipment is aboard an escort. Better take off and wait for it to arrive.`
+
+
+mission "Journalism 3"
+	landing
+	name "Slave trade"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction."
+	minor
+	source "Maker"
+	destination "Hippocrates"
+	infiltrating
+	to offer
+		has "Journalism 2: done"
+	passengers 1
+	cargo "stealth drones" 1
+	on offer
+		conversation
+			`You drop off Ale and River at the spaceport and wish them luck. They are visibly embarrassed to be in security uniforms or perhaps uncomfortable with their disguises which make them virtually unrecognizable. Bunker can't stop laughing. Everyone exchanges wishes of good luck and they walk off into the spaceport. You and bunker go back to the <ship> and prepare to launch.`
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+
+mission "Journalism 4"
+	landing
+	name "Slave trade"
+	description "Pick up the team on <destination>."
+	minor
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Journalism 3: done"
+	passengers 1
+	cargo "stealth drones" 1
+
+	on offer
+		conversation
+			`It's not hard to find a remote canyon within range of the target area. You stay hovering in case you need a quick escape but are well inside the atmosphere so Bunker releases the drones from the open cargo bay without a pressure suit. They are so small you can't make them out from the cockpit, nor can you see them on your ships scanners. When Bunker returns he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`
+			`	Once the drones are on site you can see a lone syndicate ship with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
+			`	He hums to himself for awhile before making small talk, "You know, there's a part of me that would much rather be liberating slaves instead of filming them. But for now it seems like this is the best way for me to help. Say we did liberate them right now. There is just going to be another slave ship tomorrow. But if we create political and social will; that could stop the slave ships forever."`
+			choice
+				`	"I think you're right."`
+				`	"What about doing both?"`
+					goto plan
+			`	"And you know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`
+					goto showtime
+			label plan
+			`	"Now you're talking! But it would take a lot of planing to do it without killing the people we are trying to liberate. You know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`
```

Comment:
```suggestion
			`	"Now you're talking! But it would take a lot of planing to do it without killing the people we're trying to liberate. You know they'll use those poor people as human shields. Say we-" he is cut off by the proximity alert.`
```